.github/workflows/aws-test-frontend-role.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  release:
+    types: [created]
+
+name: Test Role
+
+jobs:
+  deploy:
+    name: Test Frontend Role Configuration
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      
+    - name: Get bot token from Parameter Store 
+      uses: marvinpinto/action-inject-ssm-secrets@latest 
+      with: 
+        ssm_parameter: /frontend-team/github-actions/parameters/AWS_FRONTEND_NONPROD_ROLE
+        env_variable_name: AWS_FRONTEND_NONPROD_ROLE  
```

Comment:
Would you need to configure AWS credentials before this as well? It seems like it would need to know the access keys, region, etc.

Reply:
Should be ok now.